src/interfaces.ts

```diff
@@ -63,4 +63,5 @@ export interface groupInterface {
 export interface regionInterface {
     id: number;
     name: string;
+    cities: cityInterface[];
```

Comment:
Har vi fortsatt regions?
Trodde regions bare var ettfelt i City?

Reply:
Jeg bruker fortsatt regionInterface for å rendere regionCard. Jeg sorterer alle byene fra fetch/søk i regionene dvs. lister av byer i hver region. så passer jeg den lista inn i regioncard som renderer egne cityCards for hver by som blir passert videre